examples/evaluations/text_generation/counterfactual_metrics_demo.ipynb

```diff
@@ -456,13 +456,13 @@
    ],
    "source": [
     "# Check for fairness through unawareness\n",
-    "attribute = 'race'\n",
     "df = pd.DataFrame({'prompt': prompts})\n",
```

Comment:
I think we can remove this line, `check_ftu` method expect prompts to be a list of strings.

Reply:
I just updated the demo notebook as well